src/reverse.ts

```diff
@@ -1,5 +1,19 @@
 import { purry } from './purry';
 
+// prettier-ignore
```

Comment:
FYI, prettier keeps adding brackets around `(...infer L)`, and typescript doesn't like that ... updating prettier to the latest version didn't fix this, so I didn't do that. I will try to find an open issue in `prettier` or create one if I don't find anything

Reply:
So I'm pretty sure this is the issue: https://github.com/prettier/prettier/issues/9041

It has been solved already, and it should actually be fixed with 2.1.1: https://github.com/prettier/prettier/blob/master/CHANGELOG.md#fix-broken-format-for-infer-t-9044-by-fisker

I'll try uprading again

Reply:
right, I actually just needed to restart my editor to make it pick up the new prettier 🙄 .

upgraded and re-formatted here: 48cc6c1e